[2023-01-23 20:53:09,719] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: ml-workflow.transition_best_model_version_to_prod_task 2023-01-23T20:51:41.898006+00:00 [queued]>
[2023-01-23 20:53:09,738] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: ml-workflow.transition_best_model_version_to_prod_task 2023-01-23T20:51:41.898006+00:00 [queued]>
[2023-01-23 20:53:09,738] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-01-23 20:53:09,738] {taskinstance.py:1043} INFO - Starting attempt 1 of 3
[2023-01-23 20:53:09,738] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-01-23 20:53:09,768] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): transition_best_model_version_to_prod_task> on 2023-01-23T20:51:41.898006+00:00
[2023-01-23 20:53:09,788] {standard_task_runner.py:52} INFO - Started process 205357 to run task
[2023-01-23 20:53:09,802] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'ml-workflow', 'transition_best_model_version_to_prod_task', '2023-01-23T20:51:41.898006+00:00', '--job-id', '29', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/modeling_dag.py', '--cfg-path', '/tmp/tmplwwuccdo', '--error-file', '/tmp/tmpbnp2wqpz']
[2023-01-23 20:53:09,809] {standard_task_runner.py:77} INFO - Job 29: Subtask transition_best_model_version_to_prod_task
[2023-01-23 20:53:09,870] {logging_mixin.py:104} INFO - Running <TaskInstance: ml-workflow.transition_best_model_version_to_prod_task 2023-01-23T20:51:41.898006+00:00 [running]> on host 3f7900fce849
[2023-01-23 20:53:09,943] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=abdessamad
AIRFLOW_CTX_DAG_ID=ml-workflow
AIRFLOW_CTX_TASK_ID=transition_best_model_version_to_prod_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-23T20:51:41.898006+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-23T20:51:41.898006+00:00
[2023-01-23 20:53:10,146] {logging_mixin.py:104} WARNING - 2023/01/23 20:53:10 WARNING mlflow.tracking.context.git_context: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh()

All git commands will error until this is rectified.

This initial warning can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|none|n|0: for no warning or exception
    - warn|w|warning|1: for a printed warning
    - error|e|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2023-01-23 20:53:10,146] {logging_mixin.py:104} WARNING - 
[2023-01-23 20:53:10,331] {python.py:118} INFO - Done. Returned value was: None
[2023-01-23 20:53:10,351] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=ml-workflow, task_id=transition_best_model_version_to_prod_task, execution_date=20230123T205141, start_date=20230123T205309, end_date=20230123T205310
[2023-01-23 20:53:10,388] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-01-23 20:53:10,427] {local_task_job.py:146} INFO - Task exited with return code 0
